bbn-test-4/finality-providers/README.md

```diff
@@ -107,8 +121,21 @@ stakercli transaction create-phase1-staking-transaction \
 }
 ```
 
-After signing the transaction you should have it in hex format (see output above).
-This format is ready for propagation to the Bitcoin ledger,
+After signing the transaction you should have it in
+[hex format](https://github.com/babylonchain/btc-staker/blob/ebb2e845d7be835c7d7f7e0347600092e5de67a3/docs/create-phase1-staking.md#sign-transaction).
+It is possible to verify if your transaction has the correct parameters before
+submit the transaction to BTC chain by using the following script:
+
+```shell
+export FP_BTC_PK=<your_fp_pk>
+export SIGNED_TX=<your-signed-tx-hex>
+
+./bbn-test-4/finality-providers/scripts/fp-check-tx.sh
+
+Provided transaction is valid staking transaction!
+```
+
+This signed tx in hex format is ready for propagation to the Bitcoin ledger,
 which can happen in several ways:
```

Comment:
```suggestion
The success of the above command means that the signed transaction in hex format
is ready for propagation to the Bitcoin ledger.
This can happen in several ways:
```

Reply:
https://github.com/babylonchain/networks/pull/193/commits/a382b6949d5b7343ecc998a38f233aa8b156dd15